scripts/e2e.ts

```diff
@@ -104,7 +110,6 @@ class E2ETestRunner extends events.EventEmitter {
       if (Object.keys(deploymentProfiles.defaults).length > 0 || Object.keys(deploymentProfiles.locked).length > 0) {
         throw new Error("Trying to run e2e tests with existing deployment profiles isn't supported.");
       }
-      process.env.NODE_ENV = 'test';
```

Comment:
This means we'll end up with `NODE_ENV` being `production` (because it's otherwise unset, per `build-utils.ts`) in the test? (It's not clear from the commit message what else sets it.)

Note that we still check for `NODE_ENV` containing `test` in `lima.ts`.

Reply:
This is addressed in #5358